### PR TITLE
add modal component

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

File: pages/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.sui_modal_wrapper {
+  width: 100vw;
+  height: 100vh;
+  z-index: 999;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: var(--BLACK-C);
+  display: none;
+}
```
Comment (dev-prabhat) [inline pages/components/modal/modal.css:10]: try to give spacing between two CSS classes

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

**Review by itsmebipul00 — APPROVED** [review]

**Review by purohitdheeraj — APPROVED** [review]
All looking Good for merge 👍

**Review by zoef-yt — COMMENTED** [review]
Looks good to me. Good to merge.

**Review by zoef-yt — APPROVED** [review]

File: pages/components/modal/modal.css (hunk 1 above)
Comment (iRohitGaur) [reply pages/components/modal/modal.css:10]: I have it this way everywhere. Consistently. Personal preference.

**Pull request merged**